### ISSUE TITLE
Mixcloud stream stop playing on MPD update

### ISSUE THREAD

**Issue by marcinkruk** [patrickkfkan/volumio-mixcloud#1]
Hey, first of all thanks a lot for a mixcloud plugin for volumio, I've been searching for such solution for some time.

I started using the plugin on my device, but shows stop playing every 10 minutes or so. I looked into volumio log via volumio.local/dev api and found out this issue happens on an MPD state update. Revelant log below.

I'll try to look into the issue in more detail, any ideas on other possible causes?

```
info:
---------------------------- MPD announces state update: player
info: ControllerMpd::getState


info: ControllerMpd::pushState
info: CoreCommandRouter::servicePushState
info: CoreStateMachine::pushState
info: CoreCommandRouter::volumioPushState
info: CoreCommandRouter::executeOnPlugin: volumiodiscovery , saveDeviceInfo
info: CorePlayQueue::getTrack 0


info: CoreStateMachine::syncState   stateService stop
info: CoreStateMachine::syncState   currentStatus play
info: CoreCommandRouter::volumioPushState
info: CoreCommandRouter::executeOnPlugin: volumiodiscovery , saveDeviceInfo
info: CoreStateMachine::stPlaybackTimer
info: ------------------------------ 15ms
```

**Comment by patrickkfkan** [comment]
Another user reported this issue on Volumio forums, so this is something to look into. Because this doesn't happen on my Volumio devices (running 2.853 + [custom MPD 0.21.16](https://community.volumio.org/t/mpd-0-21-16-for-volumio-arm-armv7-and-x86/11554), I would need to know the exact version of Volumio you're running, and the device you're running it on.

**Comment by marcinkruk** [comment]
Volumio v. 2.861, no MPD modification, running on Raspberry pi 2

**Comment by marcinkruk** [comment]
Do you prefer to move this discussion to the volumio forums?

**Comment by patrickkfkan** [comment]
Yes, let's carry on the discussion there (but don't close this issue until it's really resolved).

**Comment by patrickkfkan** [comment]
Closed with commit 23c98e6
